apps/datadogcharts/datadog_charts.star

```diff
@@ -174,7 +195,7 @@ def redener(display_name, datapoints):
 def get_schema():
     dd_site_options = [
         schema.Option(
-            display = "US",
+            display = "US1",
```

Comment:
Note that these changes can break existing installations of the app. One way around that is to add a translation step in code. We unfortunately don't have a way of automatically migrating existing app configuration yet.

Reply:
Ah, ok. Just fixing some confusion but we can remove this line for now.

Reply:
Oh, actually, this one isn't a problem. =) But the ones changing the value attribute would be.